src/main/java/com/codeborne/selenide/LocalStorageConditions.java

```diff
@@ -0,0 +1,15 @@
+package com.codeborne.selenide;
+
+import java.util.function.Predicate;
+
+public class LocalStorageConditions {
+
+  public static Predicate<LocalStorage> itemPresent(String item) {
+    return localStorage -> localStorage.containsItem(item);
+  }
+
+  public static Predicate<LocalStorage> itemHasValue(String item, String value) {
+    return localStorage -> localStorage.getItem(item).equals(value);
+  }
+
+}
```

Comment:
ditto

Reply:
Here is no polling, only predicate. What are you mean?

Reply:
Opps, miss comment